src/plugins/intel_cpu/src/memory_desc/blocked_memory_desc.h

```diff
@@ -4,11 +4,21 @@
 
 #pragma once
 
+#include <bitset>
+
 #include "cpu_memory_desc.h"
 
 namespace MKLDNNPlugin {
 
+#define BLOCKED_DESC_FULL_MASK 0xffffffff
+#define BLOCKED_DESC_EMPTY_MASK 0x0
+#define BLOCKED_DESC_SKIP_OFFSET_MASK 0x7fffffff
+#define BLOCKED_DESC_OFFSET_MASK_POS 31
+
 class BlockedMemoryDesc : public virtual MemoryDesc {
```

Comment:
At the moment I think we do not need to extract the offset flag from the bitmask, simply because it will not fundamentally change anything. In other words, what is the difference between checking 31st bit in the bitmask and offset flag? The only reason could be if the other bits are senseless, but it is not the case at least for the blocked descriptor.  But probably it will make sense if we extract the offset parameter higher in the hierarchy.

Reply:
`In other words, what is the difference between checking 31st bit in the bitmask and offset flag?`
no difference. for me it would look more clear if we use mask only for strides